src/main/pages/user-management/authentication.md

```diff
@@ -10,39 +10,69 @@ folder: user-management
 
 ## Authentication on Che Master
 
-### Auth Filters on master
+### OpenId
+
+   OpenId authentication on Che master, implies presence of an external OpenId provider and has 2 main steps:
+   1. Authenticate the user through the JWT token he brought or redirect him to login;
+  
+        (Authentication tokens should be send in a `Authorization` header. Also, in limited cases when it's not possible to use `Authorization` header, 
+        token can be send in `token` query parameter. 
+        An example of such exceptional case can be: OAuth authentification initialization, IDE shows javadoc in iframe where authentication must be initialized.)
+   2. Compose internal “subject” object which represents the current user inside of the Che master code.
+  
+  At the time of writing the only supported/tested OpenId provider is Keycloak, so all examples/links will refer to this implementation. 
+
+ The flow starts from the settings service where clients can find all the necessary URLs and properties of the OpenId provider such as `jwks.endpoint`, `token.endpoint`, `logout.endpoint`, `realm.name`, `client_id` etc returned. in JSON format.
+ Service class is **org.eclipse.che.multiuser.keycloak.server.KeycloakSettings**, and it is bound only in multi-user version of Che,
+ so by its presence it is possible to detect is authentication enabled in current deployment or not. 
```

Comment:
looks like typo and **if** should be instead of **is** in `to detect is authentication enabled`